src/test/resources/com/atomist/rug/test/gherkin/handler/event/PassingFeature2Steps.ts

```diff
@@ -0,0 +1,14 @@
+import {Given,When,Then, HandlerScenarioWorld} from "@atomist/rug/test/handler/Core"
+import * as node from "../../../handlers/event/Nodes"
+
+When("i call you", world => {
+   world.registerHandler("ReturnsAMessage")
+   world.sendEvent(new node.Commit)
+})
+Then("you call me", world => {
+    return world.message() != null
```

Comment:
I think that rather than having to know to call world.messag(), we should be able to call world.plan(), and have the message put inside a plan. This is consistent with the way Rug works, and means that when testing, we don't need to know if a message is returned or a plan. The interface for all Handlers (Scala side) returns Option[Plan], so I think we should be consistent.